src/components/ReviewCard.jsx

```diff
@@ -25,27 +30,13 @@ const ReviewCard = ({ data }) => {
   return (
     <div>
       {!showIndvReview ? (
-        <ul className="review-card">
-          {data.reviews.map((review) => (
-            <li
-              key={review.review_id}
-              onClick={() => handleClick(review.review_id)}
-            >
-              <h3>{review.title}</h3>
-              <h4>Designed by {review.designer}</h4>
-              <p>owner: {review.owner}</p>
-              <img src={review.review_img_url} alt={review.title} />
-              <p>Category: {review.category}</p>
-              <p>
-                Created at {new Date(review.created_at).toLocaleString("en")}
-              </p>
-            </li>
-          ))}
-        </ul>
+        <ReviewCardReusable data={data} handleClick={handleClick} />
```

Comment:
Lets call this ReviewCardData, ReviewCardReusable doesn't fit as it can be easily confused with ReviewCard